style.css

```diff
@@ -91,6 +92,38 @@ border-radius: 50%;
 .menu__item:last-child {
   margin-right: 0;
 }
-h1 {
-  font-family: 'Roboto', sans-serif;
+.main {
+  display: block;
+  align-items: center;
+  padding-right: 63%;
+  background: top right no-repeat url(img/mainImg.svg);
+}
+.main__title {
+  margin: 0 0 40px 0;
+  margin-bottom: 40px;
+  font: 50px/65px pt_serif,arial,helvetica,sans-serif;
+}
+.main__title-text_highlight {
+  background: #ecf3ff;
+  font: 50px/65px pt_serif,arial,helvetica,sans-serif;
+}
+.main__subtitle {
+  margin: 0 0 41px 0;
+  margin-top: 30px;
+  margin-bottom: 41px;
+  font: 18px/21px Roboto,arial,helvetica,sans-serif;
```

Comment:
Codacy found an issue: [Expected single space after "," in a single-line list (value-list-comma-space-after)](https://app.codacy.com/gh/WebHeroSchool/HTML_CSS_2.0-Artem1998/pullRequest?prid=6242157)